finagle/h2/src/main/scala/io/netty/handler/codec/http2/H2FrameCodec.scala

```diff
@@ -298,6 +298,22 @@ object H2FrameCodec {
         ctx.fireChannelRead(data)
         0 // bytes are marked as consumed via WindowUpdateFrame writes
       }
+
+      override def onPingRead(
```

Comment:
do we need this? we never need to handle pings ourselves, right?

Reply:
That's right. I think this must have been left over code.